CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+# require at least cmake v3.0.0, so we can use CTest
+cmake_minimum_required(VERSION 2.8...3.13)
+
+# compile flags: -Wall "enable all compiler warnings"
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+
+# c++ language version level: c++11
+set(CMAKE_CXX_STANDARD 11)
+# cmake project name: "project1"
+project(libsat355 VERSION 0.1.0)
+
+# add CTest capability to "cse340"
```

Comment:
no longer "cse340"
Q: is this how it should be tested? using CTest?
I don't know anything about CTest, perhaps its worth investigating